dask_jobqueue/jobqueue.yaml

```diff
@@ -3,12 +3,13 @@ jobqueue:
     name: dask-worker
 
     # Dask worker options
-    threads: 2
-    processes: 4
-    memory: 8GB
-    interface: null
-    death-timeout: 60
-    local-directory: null
+    cores: null                 # Total number of cores per job
+    memory: null                # Total amount of memory per job
```

Comment:
Don't like the memory default to null and thus None in the code. But it is much more difficult to set to a defaut value when it is a global one. 
I would give default core and memory values here, or fail properly if memory is None, as for cores. Values are tightly linked to each other.

Reply:
If we choose defaults then what should they be?  

I'd prefer to raise with informative errors if memory isn't specified.  I'd be happy to make that change.

Reply:
Done